packages/ai-bot/helpers.ts

```diff
@@ -28,7 +29,12 @@ type TextMessage = {
   content: string;
 };
 
-export type Message = CommandMessage | TextMessage;
+type Break = {
+  type: ParsingMode.Break;
+  content: null;
+};
```

Comment:
What mode is Break supposed to be??

Reply:
This was meant to indicate a point when the parsing switched from text to command so that the bot could quickly return the finished message.

It wasn't clear so I've
* Removed the enum, replaced with just text (as per discussion with Ed)
* Removed the ParsingMode naming as it's not really communicating what it's now doing
* Removed the break and just started marking the text messages as complete or not. When they're not complete, the bot buffers and sends larger chunks. When they're marked as complete it sends them regardless.